packages/extension-huggingface/README.md

```diff
@@ -94,3 +96,60 @@ SELECT {{ products.value() | huggingface_table_question_answering(query=question
   }
 ]
 ```
+
+### Text Generation
+
+The [Text Generation](https://huggingface.co/docs/api-inference/detailed_parameters#text-generation-task) is one of the Natural Language Processing tasks supported by Hugging Face.
+
+Using the `huggingface_text_generation` filter. The result will be a string from `huggingface_text_generation`.
+
+**Notice**: The **Text Generation** default model is **gpt2**, If you would like to use the [Meta LLama2](https://huggingface.co/meta-llama) models, you have two method to do:
```

Comment:
"If you would like to use the [Meta LLama2] models, you have two method to do"
Check the grammar.

Reply:
Thanks for finding the grammar issue, I have fixed the `method` to `methods`.